nih-sparc/sparc-app
# Format dataset citations - Updated style

# Description

- Style updates to the citation notifications, tabs, and button. ([Designs](https://app.abstract.com/share/f93d41ce-bfd5-4b25-b8f6-18afd7f06036))
- Created mixin so notifications can be easily used elsewhere, as they require a little bit of element-ui magic to setup in the desired style. The mixin will handle setting up the style, all that needs to be passed in is the message.  

## Type of change


# How Has This Been Tested?

1. Navigate to the Find Data page.
2. Select any dataset.
3. Go to the about tab.
4. Click copy citation button.
5. Notification will appear with correct style on the top of the screen for 5 seconds notify of successful copy.
6. Select a different citation type.
7. Repeat the steps 4 & 5.


# Checklist:
